gno.land/cmd/gnoland/node_id.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/gnolang/gno/tm2/pkg/p2p"
+)
+
+// Display a node's persistent peer ID to the standard output.
+func newNodeIDCmd(bc baseCfg) *commands.Command {
+	cmd := commands.NewCommand(
+		commands.Metadata{
+			Name:       "node",
```

Comment:
As mentioned in the "validator" subcommand, I propose combining "node" and "validator" into one command, possibly called "info" or "node".